src/app/components/account/account-list-item-layout.tsx

```diff
@@ -17,6 +21,10 @@ interface AccountListItemLayoutProps extends StackProps {
   accountName: React.ReactNode;
   avatar: React.ReactNode;
   balanceLabel: React.ReactNode;
+  hasCopied?: boolean;
+  hasCopiedBtc?: boolean;
+  copyToClipboard?(e: React.MouseEvent): void;
+  copyBtcToClipboard?(e: React.MouseEvent): void;
```

Comment:
can we use `on` prefixed naming? this makes it more clear its an event

Reply:
done